src/analysis/ReachingDefinitions/ReachingDefinitions.h

```diff
@@ -39,14 +51,17 @@ enum class RDNodeType {
         // return from the call (in caller)
         CALL_RETURN,
         // dummy nodes
-        NOOP
+        NOOP,
+        // nodes that use the pointer (not necessarily the memory)
```

Comment:
Should it be just a use of a memory?

Reply:
yes, the comment is wrong as I fixed the GEP instruction. will fix in appropriate commit